flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/TaskExecutorResourceUtils.java

```diff
@@ -549,9 +558,13 @@ private static void sanityCheckShuffleMemory(final Configuration config, final M
 			}
 			if (isShuffleMemoryFractionExplicitlyConfigured(config) &&
 				!derivedShuffleMemorySize.equals(totalFlinkMemorySize.multiply(shuffleRangeFraction.fraction))) {
-				throw new IllegalConfigurationException("Derived Shuffle Memory size("
-					+ derivedShuffleMemorySize.toString() + ") does not match configured Shuffle Memory fraction ("
-					+ shuffleRangeFraction.fraction + ").");
+				LOG.info(
```

Comment:
Should we log this as a warning?

Reply:
We always had this contract for shuffle memory fraction and it is documented. I think the warning message is too severe for this notification. There is nothing really wrong about it.